minijinja-py/src/environment.rs

```diff
@@ -12,15 +12,16 @@ use crate::typeconv::{
 };
 
 /// Represents a MiniJinja environment.
-#[pyclass(subclass)]
+#[pyclass(subclass, module = "minijinja._lowlevel")]
 pub struct Environment {
     inner: Mutex<minijinja::Environment<'static>>,
 }
 
 #[pymethods]
 impl Environment {
     #[new]
-    fn py_new() -> PyResult<Self> {
+    #[pyo3(signature = (**_kwargs))]
+    fn py_new(_kwargs: Option<&PyDict>) -> PyResult<Self> {
```

Comment:
I had to add this `_kwargs: Option<&PyDict>` to make it work on Python 3.11, otherwise

```bash
$ python hello.py
Traceback (most recent call last):
  File "/Users/messense/Projects/minijinja/minijinja-py/hello.py", line 20, in <module>
    env = Environment(templates={
          ^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Environment.__new__() got an unexpected keyword argument 'templates'
```

I'm not sure why, maybe @davidhewitt can shed some light?

Reply:
Add a `__new__` that ignores the extra arguments to `minijinja.Environment` also works

```python
class Environment(_lowlevel.Environment):
    """Represents a MiniJinja environment"""
    def __new__(cls, *args, **kwargs):
        return super().__new__(cls)
```

Reply:
It's from upgrading PyO3 to 0.18, there was a bug where new which takes no arguments would not error if arguments are passed. I can see it might be subtly annoying in inheritance cases like this!